client/tsconfig.json

```
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "strict": true,
+    "jsx": "preserve",
+    "importHelpers": true,
+    "moduleResolution": "node",
+    "allowJs": true,
```

Do you guys also see this error in VsCode if you open this file?

```
Cannot write file '/home/jake/projects/VIAME-Web/client/src/common/viame.service.js' because it would overwrite input file.
```

It seems to stem from the fact that we're using the `allowJs` option. This seemed to fix it for me:

```suggestion
    "allowJs": true,
    "outDir": "./dist",
```